core/store/orm/locking_strategies_test.go

```diff
@@ -104,11 +104,16 @@ func TestPostgresLockingStrategy_WhenLostIsReacquired(t *testing.T) {
 
 	delay := store.Config.DatabaseTimeout()
 
+	// NewStore no longer takes a lock on opening, so do something that does...
+	err := store.ORM.RawDB(func(db *gorm.DB) error {
```

Comment:
RawDB is left over from when `db` was private I think.

You can simply call `store.DB` now instead. Probably should remove `RawDB` because its a bit unnecessary to go through that step.

Unless `RawDB` takes the lock now?

Reply:
Is there a way to simply "get lock". Like a function or something?

Reply:
Just gonna call it `RawDBWithAdvisoryLock` for now, will be getting rid of it soon.